swarm/runtime/statsdb/rebuild.py

```
@@ -278,43 +293,45 @@ def rebuild_stats_db(
+                                        db.record_file_change(
+                                            run_id=run_id,
+                                            step_id=step_id,
+                                            file_path=fc.get("path", ""),
+                                            change_type=fc.get("status", "modified"),
+                                            lines_added=fc.get("insertions", 0),
+                                            lines_removed=fc.get("deletions", 0),
+                                        )
+
+                                stats["envelopes_processed"] += 1
+
+                            except (json.JSONDecodeError, IOError) as e:
```

Ruff (0.15.0)

[warning] Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)